### DISCUSSION TITLE
2021.02.18 목 회의록

### DISCUSSION THREAD

**Discussion by hakju** [hakju/ios-cloud-notes#12] [General]
### [ 계획 ]

1. 오늘 수업예습
2. TIL 꼭 쓰기
3. Size class 공부 - https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/
4. ARC 조금 더 학습하기 (붱이랑 오늘 이야기하기)

**Comment by soleJin** [comment]
+ 수업 예습
+ TIL작성
+ Size Classes 공부
